eng/Publishing.props

```
@@ -1,154 +1,108 @@
-      are published from every job. RID-agnostic nupkgs are built with the same ID/version by
-      every job, so one specific job's outputs must be picked to sign and publish.
-    -->
-
-    <!-- RID-specific framework packs. -->
-    <RuntimeNupkgFile
-      Include="
-        $(DownloadDirectory)**\Microsoft.*.Runtime.*.nupkg;
-        $(DownloadDirectory)**\Microsoft.*.App.Host.*.nupkg;
-        $(DownloadDirectory)**\Microsoft.*.App.Crossgen2.*.nupkg"
-      Exclude="@(DownloadedSymbolNupkgFile)" />
-
-    <!-- VS insertion packages, carrying RID-specific installers. -->
-    <RuntimeNupkgFile
-      Include="$(DownloadDirectory)**\VS.Redist.Common.*.nupkg"
-      Exclude="@(DownloadedSymbolNupkgFile)" />
 
     <!--
-      Workloads VS insertion artifacts produced by src/workloads/workloads.csproj. Only grab
-      the zip artifacts as they're grouped by SDK feature band which correlates with specific VS versions.
-    -->
-    <WorkloadsVSInsertionFile Include="$(DownloadDirectory)*\workloads-vs\**\*.zip" />
+      By default, we don't use the default artifacts settings for publishing (only for signing).
+      Every job will publish their RID-specific packages.
+      For non-RID-specific packages, we have various rules:
 
-    <!--
-      Runtime packages associated with some identity packages. Need to exclude "runtime.native.*"
-      because Libraries produces some "runtime.native.Foo" packages with
-      "runtime.<rid>.runtime.native.Foo" identity packages.
-    -->
-    <RuntimeNupkgFile
-      Include="$(DownloadDirectory)**\runtime.*.nupkg"
-      Exclude="
-        $(DownloadDirectory)**\runtime.native.*.nupkg;
-        @(DownloadedSymbolNupkgFile)" />
+      - A job can specify EnableDefaultArtifacts=true as a global property to publish all packages it produces.
+        We have specific jobs that produce RID-agnostic packages or packages for multiple RIDs set this property.
+      - For some target RIDs, we also include specific RID-agnostic packages.
 
-    <!--
-      Packages that aren't matched above as RID-specific are considered RID-agnostic. Also include
-      the packages from the Libraries build.
+      Packages that do not meet the above rules are added with Vertical visibility in the VMR and excluded in non-VMR builds.
     -->
-    <RidAgnosticNupkgToPublishFile
-      Include="
-        $(DownloadDirectory)**\Microsoft.NET.Workload.Mono.Toolchain.*Manifest-*.nupkg;
-        $(DownloadDirectory)**\Microsoft.NET.Sdk.WebAssembly.Pack.*.nupkg;
-        $(DownloadDirectory)*\$(PublishRidAgnosticPackagesFromPlatform)\**\*.nupkg;
-        $(DownloadDirectory)*\Libraries_WithPackages\**\*.nupkg"
-      Exclude="@(RuntimeNupkgFile);@(DownloadedSymbolNupkgFile)" />
-
-    <TransportPackagesToPublishFile
-      Include="$(DownloadDirectory)**\*Transport*.nupkg"
-      Exclude="@(RuntimeNupkgFile);@(RidAgnosticNupkgToPublishFile);@(DownloadedSymbolNupkgFile)" />
+    <EnableDefaultArtifacts>false</EnableDefaultArtifacts>
```

Can we set the default that is currently set in DotNetBuild.props directly here?

```suggestion
    <EnableDefaultArtifacts Condition="'$(DotNetBuildOrchestrator)' != 'true'">false</EnableDefaultArtifacts>
```

... and pass false in, from DotNetBuild.props when doing a short stack build.